### ISSUE TITLE
NLP improvements

### ISSUE THREAD

**Issue by jpfairbanks** [jpfairbanks/SemanticModels.jl#108]
1. add column to CSV output that identifies source for each extracted fact (ie which chapter it came from SIS, SEIR, etc)
2. try lemmatizing s/o and compare if they are the same term (ie model vs the models) and if so, remove from graph
3. need a solution for entity linking ... start by post-processing the CSV (ie remove articles, lowercase, etc) so that equivalent nodes are represented as a single node
-Natalie

Ref #96

**Comment by kuncao0228** [comment]
added to the nlpextraction branch processSVOVertices.py that creates a sim_file.csv that contains the similarity for both the source vs source nodes and target vs target nodes for thresholds above .85

**Comment by kuncao0228** [comment]
a lot of the target nodes have extremely high thresholds for cases like There vs there, and Model vs model, I'm thinking of merging edges for these cases and creating the similarity edge for the other cases we talked about

**Comment by jpfairbanks** [comment]
yeah that makes sense. To have two thresholds eg, .98+ means merge the vertex .80+ means create a "is similar to" edge

**Comment by crherlihy** [comment]
@kuncao0228 @jpfairbanks  seems like you resolved this but I think there might be a case for creating a list of stopwords, and/or lower-casing non-proper nouns so that terms which are actually identical can be de-duped rather than merged. Also worth documenting somewhere non-code based what our cutoff thresholds were and why we chose them.

**Comment by jpfairbanks** [comment]
Yeah i think we will need to create a set of action items for how to improve the quality of our NLP pipeline over the next year.
